moved/src/genesis/mod.rs

```diff
@@ -5,13 +5,21 @@ use {
 
 pub use framework::FRAMEWORK_ADDRESS;
 
+mod cache;
 pub mod config;
 mod framework;
 mod l2_contracts;
 
 pub fn init(
     config: &GenesisConfig,
     state: &impl State<Err = PartialVMError>,
+) -> (ChangeSet, TableChangeSet) {
+    cache::try_load().unwrap_or_else(|| cache::save(config, state))
```

Comment:
Should we feature block cached data to tests only?

Reply:
I agree this feels like a tests-only implementation.